lib/src/generator/generator_utils.dart

```diff
@@ -34,6 +34,16 @@ String generateCategoryJson(Iterable<Categorization> categories, bool pretty) {
   return encoder.convert(indexItems.toList());
 }
 
+String removeHTMLTags(String? input) {
+  if (input == null) {
+    return '';
+  }
+  RegExp exp = RegExp(r"<[^>]*>", multiLine: true, caseSensitive: true);
+  String beforeParsing = input.toString();
+  String parsedString = beforeParsing.replaceAll(exp, '');
```

Comment:
I don't think `input.toString()` is doing anything necessary, is it? We can just return `input.replaceAll(exp, '');`.

Reply:
Addressed in new commit